execution_workflows/GETUTR/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:20.04
+
+MAINTAINER Gregor Rot <gregor.rot@gmail.com>
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=Europe/Zurich
```

Comment:
I don't think we are specifying the timezone for the docker containers

Reply:
Ops, was automatic from another Dockerfile I had, of course we can remove, should I remove?